lib/Dialect/Moore/MooreTypes.cpp

```diff
@@ -115,6 +116,7 @@ std::optional<unsigned> PackedType::getBitSize() const {
   return TypeSwitch<PackedType, std::optional<unsigned>>(*this)
       .Case<VoidType>([](auto) { return 0; })
       .Case<IntType>([](auto type) { return type.getWidth(); })
+      .Case<TimeType>([](auto type) { return 64; })
```

Comment:
Is this tested somewhere?

Reply:
Added an explicit use of `packed_to_sbv` and `sbv_to_packed` with a time type, which calls this to ensure the types match.